src/lib/ext-popup.js

```diff
@@ -0,0 +1,111 @@
+import {
+  areExtsBeingMonitored,
+  startMonitorAll,
+  stopMonitorAll,
+  viewExtPage,
+} from '../lib/ext-listen.js';
+
+export default class Popup {
+  constructor() {
+    this.startMonitorAllBtn = document.getElementById('startMonitorBtn');
+    this.stopMonitorAllBtn = document.getElementById('stopMonitorBtn');
+    this.viewActivityLogBtn = document.getElementById('actLogPage');
+    this.monitorStatusText = document.getElementById('monitorStatus');
+    this.errorMsgText = document.getElementById('errorText');
+  }
+
+  renderErrorMsg(message) {
+    this.errorMsgText.textContent = message;
+    this.errorMsgText.style.display = 'block';
+  }
+
+  renderMonitorStartedUI() {
+    this.startMonitorAllBtn.setAttribute('disabled', 'disabled');
+    this.startMonitorAllBtn.classList.add('disabled');
```

Comment:
Nit, we can avoid to add the disabled class by using the pseudo classes in the CSS (See the MDN doc page for the [:disabled](https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled) pseudo class)

Reply:
Done. :-)